paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentSheetViewModelTest.kt

```diff
@@ -720,106 +720,31 @@ internal class PaymentSheetViewModelTest {
 
     @Test
     fun `fetchPaymentIntent() should update ViewState LiveData`() {
+        val viewModel = createViewModel()
         var viewState: PaymentSheetViewState? = null
         viewModel.viewState.observeForever {
             viewState = it
         }
-        viewModel.maybeFetchStripeIntent()
-        idleLooper()
+
         assertThat(viewState)
             .isEqualTo(
                 PaymentSheetViewState.Reset(null)
             )
     }
 
     @Test
-    fun `fetchPaymentIntent() should only fetch if intent is null`() {
-        viewModel.setStripeIntent(PaymentIntentFixtures.PI_REQUIRES_PAYMENT_METHOD)
-        var viewState: PaymentSheetViewState? = null
-        viewModel.viewState.observeForever {
-            viewState = it
-        }
-        viewModel.maybeFetchStripeIntent()
-        assertThat(viewState).isNull()
-    }
-
-    @Test
-    fun `fetchPaymentIntent() should propagate errors`() = runBlocking {
-        val paymentConfiguration = PaymentConfiguration(ApiKeyFixtures.FAKE_PUBLISHABLE_KEY)
-
-        val failingStripeRepository: StripeRepository = mock()
-        whenever(
-            failingStripeRepository.getPaymentMethods(
-                any(),
-                anyString(),
-                anySet(),
-                any()
-            )
-        ).doThrow(IllegalStateException("Request Failed"))
-
-        val viewModel = createViewModel(
-            stripeIntentRepository = StripeIntentRepository.Api(
-                stripeRepository = failingStripeRepository,
-                lazyPaymentConfig = { paymentConfiguration },
-                workContext = testDispatcher,
-                Locale.US
-            )
-        )
-        var result: PaymentSheetResult? = null
-        viewModel.paymentSheetResult.observeForever {
-            result = it
-        }
-        viewModel.maybeFetchStripeIntent()
-        idleLooper()
-        assertThat((result as? PaymentSheetResult.Failed)?.error?.message)
-            .isEqualTo("Could not parse PaymentIntent.")
-    }
-
-    @Test
-    fun `fetchPaymentIntent() should fail if confirmationMethod=manual`() {
```

Comment:
I feel like this should be (and is!) tested in `StripeIntentValidatorTest` and not here.